### PR TITLE
fix step 2 issues

### LINKED ISSUE #16: Step 2 - Tutorial issues/enhancements
[ ] - add 's' to typeDef and 'r' to resolves “remove all typeDef and resolves from”
[ ] - mention what extension is being using for syntax highlighting since /* GraphQL */ before the template string is unique.
[ ] - In the Consolidate Extracted Schemas section “Since we will be defining Query types in multiple locations, we will update our checkout.typeDef.js file to extend the Query type”. The code snippet above already included `extend type Query`. We should fix this so that it’s `type Query` initially and have them add extend if that’s what we want them to do here.
[ ] - “In later tutorials, we will be Mutations and Subscriptions to our project” need to add “we will be adding* Mutations and Subscriptions”
[ ] - we should probably get them to install graphql-tools and lodash.merge before modifying schema/index.js
[ ] - need to update code snippet query from user_email: to userEmail
[ ] - “and return the same set of 4 results.” it currently returns 5

### PR DESCRIPTION
**Pull request by kevindavi5**
closes #16

### CODE REVIEW

File: docs/_pages/2.modularize-schema.md [hunk 1]
```diff
@@ -49,12 +49,12 @@ From our index.js file, we can cut and paste the contents of the gql tag definin
 file. You’ll notice that we will no longer use the gql tag. Instead, we export a single constant that is our typeDef as just a string using template string
 syntax for easy multi-line support. Many IDEs have plugins and extensions available to aid in the development of graphql types. Installing and enabling can be
 useful for development. By adding the `/* GraphQL */` identifier before our template string, our IDE can perform proper syntax highlighting if an appropriate
-extension is loaded.
+extension is loaded. Using VS Code I prefer [Apollo GraphQL](https://marketplace.visualstudio.com/items?itemName=apollographql.vscode-apollo).
```
Comment (natergj) [inline docs/_pages/2.modularize-schema.md]: would it make sense to create a new section in https://github.com/hydrateio/advanced-graphql-server-tutorial/blob/master/docs/_pages/100.resources.md and add this there, too?

File: docs/_pages/2.modularize-schema.md [hunk 2]
```diff
@@ -49,12 +49,12 @@ From our index.js file, we can cut and paste the contents of the gql tag definin
...
 
 <pre><code class="language-graphql">
 // File: src/schema/checkout/checkout.typeDef.js
 const typeDef = /* GraphQL */`
-  extend type Query {
+  type Query {
```
Comment (natergj) [inline docs/_pages/2.modularize-schema.md]: `extend` needs to be there

Comment (natergj) [reply docs/_pages/2.modularize-schema.md]: oh wait, never mind

File: docs/_pages/2.modularize-schema.md (hunk 1 above)
Comment (kevindavi5) [reply docs/_pages/2.modularize-schema.md]: yeah, I created #40 for it.

**Pull request merged**